panel/tests/pane/test_vega.py

```diff
@@ -194,8 +207,9 @@ def test_get_vega_pane_type_from_dict():
     assert PaneBase.get_pane_type(vega_example) is Vega
 
 
-def test_vega_pane(document, comm):
-    pane = pn.panel(vega_example)
+@pytest.mark.parametrize('example', [vega_example, vega_df_example])
+def test_vega_pane(document, comm, example):
```

Comment:
Ideally it would confirm that the dataframe is correctly converted to a `ColumnDataSource`.

Reply:
Also, thanks!

Reply:
Nvm, just realized that's already what this test is doing.